recipes/pySW4/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pySW4" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "b14bb605da99ffb557fe1742914207beb876602bb69576b95fc1176304b6c58a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/shaharkadmiel/pySW4/archive/v0.3.0.tar.gz
```

Comment:
`https://github.com/shaharkadmiel/pySW4/archive/v{{ version }}.tar.gz` would be better, so the version only has to be changed in one place. (A PyPI URL would be even better, just for consistency with the default behavior in conda-forge, except it seems that you haven't put 0.3.0 on PyPI?)